cmd/ddev/cmd/root.go

```diff
@@ -177,8 +182,8 @@ func init() {
 }
 
 func instrumentationNotSetUpWarning() {
-	if !output.JSONOutput && versionconstants.SegmentKey == "" && globalconfig.DdevGlobalConfig.InstrumentationOptIn {
-		util.Warning("Instrumentation is opted in, but SegmentKey is not available. This usually means you have a locally-built ddev binary or one from a PR build. It's not an error. Please report it if you're using an official release build.")
+	if !output.JSONOutput && globalconfig.DdevGlobalConfig.InstrumentationOptIn && (versionconstants.SegmentKey == "" || versionconstants.AmplitudeAPIKey == "") {
```

Comment:
This seems like it will cause confusion, as either of the keys being missing gives the same warning.  Should we remove the checks for SegmentKey in these situations?

Reply:
Well, I did not split it because it will anyway get removed soon.